src/index.js

```diff
@@ -11,6 +21,79 @@ function handleHttpErrors(res) {
 }
 
 /*---------------------------------------------*/
+/*---------- Begin Get Person By Name ---------*/
+/*---------------------------------------------*/
+
+function fillViewPersonWithDataDiv() {
+    
+    let ptag = document.createElement('p');
+    ptag.setAttribute('id', 'viewPersonWithDataPTAG');
+
+    let inputtag = document.createElement('input');
+    inputtag.setAttribute('id', 'viewPersonWithDataInputTAG');
+    inputtag.setAttribute('type', 'text');
+    inputtag.setAttribute('placeholder', 'UserName');
+
+    let buttontag = document.createElement('button');
+    buttontag.innerHTML = 'Get User';
+    buttontag.setAttribute('id', 'viewPersonWithDataButtonTAG');
+
+    let div = document.getElementById("viewPersonWithData");
+    div.innerHTML = "";
+    div.appendChild(inputtag);
+    div.appendChild(buttontag);
+    div.appendChild(ptag);
```

Comment:
This is some, to me, *hardcore* DOM manipulation. I haven't done it on this level before. Nice!